deploy/olm-catalog/3scale-community-operator/3scale-community-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -257,7 +257,7 @@ spec:
           - tenants
           verbs:
           - '*'
-        serviceAccountName: 3scale-operator
+        serviceAccountName: 3scale-community-operator
```

Comment:
I think this should be "3scale-opertor"
This is the serviceaccount of the 3scale-operator, which has not changed

Reply:
productized operator and community operator already share CRD's. ServiceAccount also should be shared? Why not different one, so they could coexist more independently?? Deployment name has also changed.

Reply:
Both operators won't be able to coexist in the same project anyway (as the APIManager resources are always the same), so if you thought it in that way I don't see any advantage of doing that. The same applies with the Deployment name.
If the reason is "just to make it more explicit at the resource level names" I'm not strongly against it. The consequence is we'll have to maintain those differences between branches.

Reply:
will change that then, less differences, the better